src/scenic/core/pruning.py

```diff
@@ -32,14 +35,37 @@ def isMethodCall(thing, method):
         return False
     return thing.method is underlyingFunction(method)
 
+def isFunctionCall(thing, function):
+    """Match calls to a given function, taking into account distribution decorators."""
+    if not isinstance(thing, FunctionDistribution):
+        return False
+    return thing.function is underlyingFunction(function)
+
 def matchInRegion(position):
-    """Match uniform samples from a `Region`, returning the Region if any."""
+    """Match uniform samples from a `Region`, returning the Region, if any,
+    and a lower and upper bound on the distance the object will be placed
+    along with any offset that should be added to the base."""
```

Comment:
Fix docstring. (1-line summary needs to be separated by a blank line and the closing triple quotes need to be on their own line.)